lib/src/constants/enums.dart

```diff
@@ -5,3 +5,5 @@
 /// Two types for the viewer: image and video.
 /// 两种预览类型：图片和视频
 enum CameraPickerViewType { image, video }
+
+enum DominantHand { left, right }
```

Comment:
We need docs here, make it as a template so the config can ref it directly.

Reply:
Could you please give me an example, as I'm not sure what to do?

Reply:
Define: https://github.com/fluttercandies/flutter_wechat_camera_picker/blob/5370cca41456bf23fa17d15f9e1c4de5ad4d1487/lib/src/constants/type_defs.dart#L66-L72
Use: https://github.com/fluttercandies/flutter_wechat_camera_picker/blob/5370cca41456bf23fa17d15f9e1c4de5ad4d1487/lib/src/constants/config.dart#L140